src/alloc/metrics.rs

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2023 Google LLC
```

Comment:
```suggestion
 * Copyright 2024 Google LLC
```

😃 made me question what year we're in.